crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -64,6 +65,11 @@ impl DynamicScene {
     ) -> Result<(), SceneSpawnError> {
         let type_registry = type_registry.read();
 
+        // Collection of entities that have references to entities in the scene,
+        // that need to be updated to entities in the world.
+        // Keyed by Component's TypeId.
+        let mut entity_mapped_entities = HashMap::default();
```

Comment:
Bikeshed: Name could be clearer imo. Maybe `scene_mappings`?

Reply:
Yeaah, that makes sense. I updated the comment too to be a little more clear, hopefully.